pkg/common/plugin/exec.go

```diff
@@ -96,10 +99,49 @@ func (e *DefaultExecutor) ExecutePlugin(ctx context.Context, pluginPath string,
 	// stdout and stderr for some reason. Ignore failures as stderr is
 	// only informational.
 	if e.Stderr != nil && stderr.Len() > 0 {
-		_, _ = stderr.WriteTo(e.Stderr)
+		scanner := bufio.NewScanner(stderr)
+		for scanner.Scan() {
+			line := scanner.Text()
+
+			switch {
+			case strings.HasPrefix(strings.ToLower(line), "info"):
+				line = strings.Replace(line, "info: ", "", -1)
+				logrus.Infof("[Plugin] %s", line)
+			case strings.HasPrefix(strings.ToLower(line), "warn"):
+				line = strings.Replace(line, "warn: ", "", -1)
+				logrus.Warnf("[Plugin] %s", line)
+			case strings.HasPrefix(strings.ToLower(line), "debug"):
+				line = strings.Replace(line, "debug: ", "", -1)
+				logrus.Debugf("[Plugin] %s", line)
+			default:
+				fmt.Fprintf(os.Stderr, "[Plugin] %s,", line)
+			}
+		}
+		// TODO: Should this be removed since the msgs are processed?
+		//_, _ = stderr.WriteTo(e.Stderr)
 	}
+
+	var obj interface{}
+	var resultsBuffer bytes.Buffer
+
+	scanner := bufio.NewScanner(stdout)
+	for scanner.Scan() {
```

Comment:
codecov bot is showing a lower test coverage compare to main. Could we extract the scan loops so they are unit testable?